src/test/runTests.ts

```diff
@@ -0,0 +1,24 @@
+import * as path from 'path';
```

Comment:
wouldn't we need to also provide a test view then? right now, there's only a tasks view in the extension.